op-challenger/flags/flags.go

```diff
@@ -93,6 +93,11 @@ var (
 		Usage:   "URLs of L2 JSON-RPC endpoints to use (eth and debug namespace required)",
 		EnvVars: prefixEnvVars("L2_ETH_RPC"),
 	}
+	L2ExperimentalEthRpcFlag = &cli.StringFlag{
+		Name:    "l2-experimental-eth-rpc",
+		Usage:   "L2 Address of L2 JSON-RPC endpoint to use (eth and debug namespace required with execution witness support)  (cannon/asterisc trace type only)",
+		EnvVars: prefixEnvVars("L2_EXPERIMENTAL_ETH_RPC"),
+	}
```

Comment:
Could we add a test for this flag in `main_test`? Just making sure it turns up in the right place in the config like the other tests there.